aptos-move/aptos-gas-schedule/src/gas_schedule/aptos_framework.rs

```diff
@@ -242,6 +242,30 @@ crate::gas_schedule::macros::define_gas_parameters!(
         [bulletproofs_per_byte_rangeproof_deserialize: InternalGasPerByte, { 11.. => "bulletproofs.per_byte_rangeproof_deserialize" }, 121],
         // Bulletproofs gas parameters end.
 
+        // Bulletproofs batch vrfy gas parameters begin.
+        // Generated at time 1738278510.3867955 by `scripts/algebra-gas/update_bulletproofs_batch_vrfy_gas_params.py` with gas_per_ns=295.16.
+        [bulletproofs_verify_base_batch_1_bits_8: InternalGas, { RELEASE_V1_27.. => "bulletproofs.verify.base_batch_1_bits_8" }, 133_731_503],
+        [bulletproofs_verify_base_batch_1_bits_16: InternalGas, { RELEASE_V1_27.. => "bulletproofs.verify.base_batch_1_bits_16" }, 195_542_132],
+        [bulletproofs_verify_base_batch_1_bits_32: InternalGas, { RELEASE_V1_27.. => "bulletproofs.verify.base_batch_1_bits_32" }, 310_976_813],
```

Comment:
This needs to be (somewhat) consistent with the cost of individually verifying a Bulletproof for a 32-bit committed value via `verify_range_proof_internal`, no?

There, the cost would be:
```
bulletproofs_base + 32 * bulletproofs_per_bit_rangeproof_verify + (some very small deserialization cost)
= 
11_794_651 + 32 * 1_004_253
=
43_930_747
```

But here the cost is `310_976_813` which is 7x larger.

Reply:
I see you've patched things and costs are now consistent!